numpy/lib/tests/test_regression.py

```diff
@@ -167,8 +167,8 @@ def test_who_with_0dim_array(self):
         try:
```

Comment:
I don't think any of the changes in this file are worthwhile - it's just a test file, so the error should never happen anyway.